components/icon/gulpfile.js

```diff
@@ -17,13 +17,12 @@ const replace = require('gulp-replace');
 const sort = require('gulp-sort');
 const svgcombiner = require('gulp-svgcombiner');
 const svgstore = require('gulp-svgstore');
-const del = require('del');
+const rimraf = require('rimraf')
```

Comment:
You should be able to revert the changes from del to rimraf. Del is being used throughout the project, no need to substitute if for rimraf at this time.

Reply:
del package doesnt export a function named del as such as per the docs. I was getting an error on build on the main branch also. that's why I used rimraf which I saw someone has added in the package.json
Docs: https://www.npmjs.com/package/del

Can you please re run the build from the main branch and check?

Reply:
Yes it was causing some failures due to the migration.